src/components/FindPark.vue

```diff
@@ -13,12 +13,16 @@
         Find a dog park near me!
       </button>
     </router-link>
-    <h3>--Or--</h3>
+    <h3 id="location-message" v-if="!this.$store.state.geolocation">
+      Turn on location services and reload the page to search for parks near
+      you!
```

Comment:
Specifically this one here, it is trying to count extra character spaces from the end of `near` to the beginning of `you` for your test. Not sure if that is how it display in your code editor though or if it is just what Github is showing.

Reply:
Lesson learned about auto formatting. Once I turned that off and adjusted the spacing, it passed.